app/src/main/java/com/example/fbufinalapp/MainActivity.java

```diff
@@ -38,6 +41,26 @@ protected void onCreate(Bundle savedInstanceState) {
         View view = binding.getRoot();
         setContentView(view);
 
+        if (ParseUser.getCurrentUser() == null){
+            ParseUser user = new ParseUser();
+            user.setUsername("Guest");
+            user.setPassword("default");
+            user.put("favorites", new ArrayList<>());
+            user.put("itineraries", new ArrayList<>());
```

Comment:
A good habit here would be encapsulating this in its own method called something like 
`createGuestUser` or something.